### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by Sameony**
Added basic and outline alerts.
image
image
image

### CODE REVIEW

File: Templates/Alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.alert {
+  cursor: pointer;
+  border-radius: 3px;
+  display: flex;
+  align-content: center;
```
Comment (nakulsharma15) [inline Templates/Alerts/alert.css:5]: Just a suggestion, if you're using this somewhere else as well, then you can extract this in a utility class.

**Review by nakulsharma15 — COMMENTED** [review]
Given a minor Suggestion. Other than that, Everything seems good!

File: Templates/Alerts/alert.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .alert {
-  width: 100%;
+  cursor: pointer;
```
Comment (vaibhav-mahalle) [inline Templates/Alerts/alert.css:2]: I like the the short and crisp commit messages. Clean code and consistent naming

**Review by vaibhav-mahalle — APPROVED** [review]
No apparent changes looks good to me.

File: Templates/Alerts/alert.css (hunk 1 above)
Comment (Sameony) [reply Templates/Alerts/alert.css:5]: i am using a flex but the align/justify properties are not really repeating so i did not create a separate utility class for it.

**Review by nakulsharma15 — APPROVED** [review]

**Pull request merged**